src/screens/Dressroom/DressroomScreen.tsx

```
@@ -264,7 +270,7 @@ export function DressroomScreen() {
         }}
         onConfirm={() => {
           if (closetFlow.selected) {
-            closetFlow.confirmDelete((id) => setClosets((prev) => prev.filter((c) => c.id !== id)));
+            closetFlow.confirmDelete((id) => setClosetList((prev) => prev.filter((c) => c.id !== id)));
```

ESLint

[error] Replace `·setClosetList((prev)·=>·prev.filter((c)·=>·c.id·!==·id))` with `⏎··············setClosetList((prev)·=>·prev.filter((c)·=>·c.id·!==·id)),⏎············`

(prettier/prettier)